hasura/graphql-engine
# console: update select font for firefox (close #3955)

### Description

### Changelog


### Affected components



### Related Issues
closes #3955

### Solution and Design
added `font-family: sans-serif;` to bootstrap overrides and react table overrides

### Steps to test and verify
*use firefox to check
1. `http://localhost:3000/data/schema/public`, click the dropdown
2. select a table and click the dropdown below filter
3. click the dropdown in the pagination

all should have sans-serif font

#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
